server/src/main/java/org/elasticsearch/action/admin/cluster/node/stats/NodesStatsRequest.java

```diff
@@ -311,11 +214,20 @@ public void writeTo(StreamOutput out) throws IOException {
         }
     }
 
+    public static Map<String, Consumer<NodesStatsRequest>> getDispatchMap() {
```

Comment:
Why is this moved to here? It seems like an implementation detail that should stay in the stats action. The request should be a opaque list of strings?